### ISSUE TITLE
NVDA Does Not Read Kindle Book Page

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3084]
**Reported by bahsieh47 on 2013-03-14 09:37**
I installed Kindle for PC With Accessibility Plugin and NVDA on my laptop running Windows 8. After installation, I got NVDA loaded, and opened a Kindle book and hit "Control R", aiming to make NVDA read the book page. Unfortunately NVDA talked a lot of different things except reading the page. Did I hit the wrong key to start reading? I will appreciate it, if there is anyone who knows the answer and is willing to share his or her experience. Thanks.
BA Hsieh

**Comment by derekriemer** [comment]
Is there a reason this ticket has the NVDA Community Website label?

**Comment by jcsteh** [comment]
Probably a mistake. Also, this is no longer relevant, since Kindle is getting a major accessibility refactor at present.
